sdk/personalizer/ai-personalizer-rest/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Personalizer client library for JavaScript
+
+Personalizer Client
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/personalizer/ai-personalizer-rest)
+- [Package (NPM)](https://www.npmjs.com/package/@azure-rest/ai-personalizer)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/ai-personalizer?view=azure-node-preview)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/personalizer/ai-personalizer-rest/samples/v1-beta/javascript)
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge, and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this package.
+
+### Install the `@azure-rest/ai-personalizer` package
+
+Install the Azure Personalizer client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/ai-personalizer
+```
+
+### Create and authenticate a `PersonalizerClient`
+
+To use an [Azure Active Directory (AAD) token credential](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
+
+After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
+As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
+can be used to authenticate the client.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
```

Comment:
This whole section talks about AAD, remove it now and add back it when the service and SDK actually support it?

Just not to mislead the customers.

@joheredi

Reply:
Added a test that uses AAD.